content/user/payment/square.md

```diff
@@ -7,16 +7,44 @@ weight: 10
 
 There are two Square modules for Zen Cart: 
 - A newer one, based on the Square Web Payments API 
-- An older one, based on SqPaymentForm, which will be deprecated in 2022
+- An older one, based on SqPaymentForm, which will be deprecated in July 2022.
```

Comment:
The previous module based on the Square Payments Form (the SqPaymentForm library) was deprecated on May 13, 2021.  Square permanently retires deprecated modules around 1 year from deprecation date.  Current communications from Square have it being retired July 2022.  At that point, it will cease to work.